packages/gatsby-plugin-google-gtag/.npmignore

```diff
@@ -0,0 +1,34 @@
+# Logs
+logs
+*.log
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directory
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
```

Comment:
What about removing all the comments and sort the directories..?

Reply:
This comes from package template - https://github.com/gatsbyjs/gatsby/blob/master/plop-templates/package/.npmignore.hbs it doesn't hurt to have comments here as this is mostly setup once and forget